README.md

```diff
@@ -145,7 +171,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+grammer09.cpp
```

Comment:
I appreciate you writing tests, but it's really a good idea to have tests where the program reports if they pass or fail. You already have the data there: I just changed line 51 to `if(results[i] != answers[i]){` (and first answers[0], which should be `true`. No deductions for this.

However, the intent of the exercise was to have you use a table to drive the state machine in your code. You wrote a kind of parser by hand, and it does use the states as validation, but as I said on Wednesday, I'm really looking for a table driven approach.

# -6